### PR TITLE
added forced password change and added checkboc to assign leave to all employees when creating leave types 

### PR DESCRIPTION
**Pull request by TalibY22**
When a new employee was added into the system the phone number was used as the password this is a big security risk in order to minimize the security risk i implemented a forced password change while checks if the employee is new if he is he will be forced to change his password in order to use the system this ensure the security risk of using phone number is reduced.Additionally this pr also adds a checkbox when creating leave types which selects all employees to assign the leave to saves time when creating leaves types such annual or compulsory.

### CODE REVIEW

File: horilla/settings.py
Comment (horilla-opensource) [inline horilla/settings.py:1]: Hi @TalibY22 ,

Please add the changes in this file to the horilla_middlewares.py inside the horilla directory to have a smooth pull for existing users.

You can make use of the MIDDLEWARE variable inside file and append the _base.middleware.ForcePasswordChangeMiddleware_ to it.

With Regards,
**Team Horilla**

Comment (TalibY22) [reply horilla/settings.py:1]: kindly a bit of clarification do i just append the middleware for forced password change?MIDDLEWARE.append("base.middleware.ForcePasswordChangeMiddleware")

Comment (TalibY22) [reply horilla/settings.py:1]: noted

Comment (TalibY22) [reply horilla/settings.py:1]: any update?

**Comment by horilla-opensource** [comment]
Hi @TalibY22 ,

Sorry for the delay. We are merging the PR, but we need some changes in the UI which we'll do.

Thank you for your efforts and contributions.

With Regards,
**Team Horilla**

**Comment by horilla-opensource** [comment]
Hi @TalibY22 ,

There is an issue in the PR. You have created the PR to the 1.0 branch, please create a PR for the master branch.

With Regards,
**Team Horilla**

**Pull request merged**